packages/controller/conf/iobroker-dist.json

```diff
@@ -49,6 +49,17 @@
             "periodComment": "by default backup every 2 hours. Time is in minutes. To disable backup set the value to 0",
             "path": "",
             "pathComment": "Absolute path to backup directory or empty to backup in data directory"
+        },
+        "jsonlOptions": {
```

Comment:
should we add comment fields like for the other attributes, e.g. ignoreReadErrors is not instantlly clear for most users

Reply:
This would be a good idea @AlCalzone ...

Reply:
done